R/bambu.R

```diff
@@ -136,24 +136,48 @@
 #'     genome = fa.file,  discovery = TRUE, quant = TRUE)
 #' @export
 bambu <- function(reads, annotations = NULL, genome = NULL, NDR = NULL,
-    opt.discovery = NULL, opt.em = NULL, rcOutDir = NULL, discovery = TRUE, 
-    quant = TRUE, stranded = FALSE,  ncore = 1, yieldSize = NULL,  
+    mode = NULL, opt.discovery = NULL, opt.em = NULL, rcOutDir = NULL, discovery = TRUE, 
```

Comment:
the mode parameter is very helpful, would be great to have pre-defined parameters for each mode documented in the readme or documentation